CONTRIBUTING.md

```diff
@@ -80,3 +80,13 @@ In this case, to fix the issues, we need to:
 - Add an space after `if` in line 34
 
 The plugin also adds a new tab in the bottom of the IDE to run Checkstyle and show errors and warnings. We recommend that you run the report and fix all issues before submitting a pull request.
+
+## File headers
+
+All files should contain the license header which can be added automatically by running:
+
+```
+./gradlew spotlessApply
+```
+
+The copyright year should be updated manually when editing existing files.
```

Comment:
Thanks for this, obviously it is needed. I *think* though that this file might be one that we share amongst all of the Micronaut repos, thus this might not be the place to make the change. I will verify that with the team, and if correct then I'll port this change over to the master file in whichever repo it lives.